scripts/tests/github/build-gh-page.spec.ts

```diff
@@ -7,7 +7,7 @@ const command = 'npx --no tsx github/build-gh-page.ts';
 describe('build-gh-page', () => {
 	process.env.NAME = 'test-branch';
 	process.env.OUT_DIR = 'tests/fixtures';
-	process.env.OWNER_NAME = 'db-ui';
+	process.env.OWNER_NAME = 'db-ui'; // TODO: Change to db-ux if we have another simple repo
```

Comment:
```suggestion
	process.env.OWNER_NAME = 'db-ux-design-system';
```

@nmerget isn't this supposed to be `db-ux-design-system`?

Reply:
we need some `gh-pages` folder it doesn't matter. But we shouldn't use the branch from this repo. We could create a "fake" repo to tests it with just one file